apps/stupidchat/stupidchat.go

```diff
@@ -0,0 +1,25 @@
+// Package stupidchat provides details for the Stupid Chat applet.
+package stupidchat
+
+import (
+	_ "embed"
+
+	"tidbyt.dev/community/apps/manifest"
+)
+
+//go:embed stupid_chat.star
+var source []byte
+
+// New creates a new instance of the Stupid Chat applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "stupid-chat",
+		Name:        "Stupid Chat",
+		Author:      "harrisonpage",
+		Summary:     "Tidbyt Messaging",
```

Comment:
If you don't mind it'd be great to rephrase this summary a little bit, just to highlight to users that this an external service not provided by Tidbyt. Perhaps something like `"Send messages to your Tidbyt"`?

Reply:
```suggestion
		Summary:     "Send messages to your Tidbyt",
```